irods_specific_implementation/utils.hpp

```diff
@@ -60,6 +61,11 @@ namespace irods::rest
 
         _response.send(http_code, msg);
     } // handle_request
+
+    inline bool to_bool(const std::string_view s){
+        return s == "1"; // we only honor "1" for this client
+    } // to_bool
+
```

Comment:
Remove or consider renaming to something like `is_set`.

`is_set` captures the intent pretty well imo.